EIPS/eip-XXXX.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Identity aggregated NFT, self soveriegn and composible identity aggregation in NFTs (SBTs included)
```

Comment:
```suggestion
eip: 7231
title: Identity aggregated NFT, self soveriegn and composible identity aggregation in NFTs (SBTs included)
```
Issuing EIP number as PR number.  Please also update the filename

Reply:
Ok, got it, thanks a lot.